types/s3-zip/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "@definitelytyped/dtslint/dt.json" }
```

Comment:
You should remove this file; it is no longer required.

Reply:
Hi Jake I am confused. This file was automatically generated when using the generator:
```bash
npx dts-gen --dt --name <my-package> --template module
```

This file is also included in the readme section of creating a [new package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#create-a-new-package)

<img width="966" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/84688214/46b6edee-f740-453a-a32f-3005f75ae022">

So I am a bit confused. Does the generation script and readme need updating?

Appreciate your help.

Thanks.

Reply:
Yes, this just changed a couple of hours ago and when we merged stuff I went and updated the PRs; you didn't do anything wrong, it's just timing.

Readme will be fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67891.

Reply:
You will want to update `dts-gen` locally to get the new layout too.

Reply:
Thanks, gotta be quick around here! I have deleted the `tslint.json` 🙌